daisy/workflow/common.go

```diff
@@ -133,6 +131,12 @@ func substitute(s reflect.Value, replacer *strings.Replacer) {
 
 		raw := f.Interface()
 		switch raw.(type) {
+		case *Workflow:
+			switch s.Interface().(type) {
+			case SubWorkflow:
```

Comment:
I want to say you should add a default case here that just does this:
substitute(e, replacer)

Not that it is use currently